model/service_logs/v1/cluster_log_type.model

```diff
@@ -42,4 +42,7 @@ class LogEntry {
 	InternalOnly Boolean
 
 	Timestamp Date
+
+	// The username that triggered the event (unless internal).
```

Comment:
I would change the description "unless internal" to "if available"

Reply:
fixed